src/components/common/text-input.tsx

```diff
@@ -23,3 +25,22 @@ export const TextInput: FunctionComponent<ButtonProp> = ({ className, placeHolde
     />
   );
 };
+
+export const DataList: FunctionComponent<ButtonProp> = ({ placeHolder, onChange, dataTestId, options, value }) => {
```

Comment:
let's make it autocomplete instead. I have many document store and searching the correct tone is a nightmare.

Reply:
Also there are cases where we wont retrieve a doc store better to leave this editable